pom.xml

```diff
@@ -189,7 +189,7 @@
 		<!--TODO: Move transport related dependencies here pending https://jira.spring.io/browse/XD-3337  -->
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream-binder-redis</artifactId>
+			<artifactId>spring-cloud-stream-binder-rabbit</artifactId>
```

Comment:
No, sorry.  It was on my local only as I was using `rabbit` binder but added that to this commit by mistake. yeah, please merge without rabbit binder. thank you.